packages/coordinator/src/assets/icons/chevron-left-small.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="12" viewBox="0 0 8 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7 1L1.61538 6L7 11" stroke="#474747" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```

Comment:
It seems that the chevrons are never been used..

Reply:
Yes because Matar moved it to the screen header, going to use it in the next PR